lrauv_ignition_plugins/worlds/buoyant_tethys.sdf

```diff
@@ -99,7 +101,17 @@
         <scene>scene</scene>
         <ambient_light>0.4 0.4 0.4</ambient_light>
         <background_color>0.8 0.8 0.8</background_color>
-        <camera_pose>0 6 6 0 0.5 -1.57</camera_pose>
+        <!-- looking at robot -->
+        <!--camera_pose>0 6 6 0 0.5 -1.57</camera_pose-->
+        <!-- looking at all science data for 2003080103_mb_l3_las.csv -->
+        <camera_pose>-50000 -30000 250000 0 1.1 1.58</camera_pose>
```

Comment:
> Is this uncommented intentionally or just for reviewing?

It was just for reviewing, but now that scrolling is actually easier, I just reverted it: 6f92329

> a future PR can add a toggle to the VisualizePointCloud panel, to automatically zoom out to the science data, calculated by the data's bounding box.

Oh interesting feature request. I think of camera controls more like something that belongs in the `ViewAngle` plugin, but I think it would be reasonable to add this to `VisualizePointCloud`.